README.md

```diff
@@ -1,2 +1,209 @@
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+
+For more information on the importance of a professional README for your repositories: https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/readme_best_practices.md
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  "Daniel O. Tagua"
+  <br/>
+
+  <h3><b>Porfolio Mobile</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Porfolio Mobile ](#-porfolio-mobile-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Porfolio Mobile <a name="about-project"></a>
+
+This project is the first part of the portfolio for mobile
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML.
+2. CSS.
+3. LINTERS.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="./index.html">HTML</a></li>
+    <li><a href="./style.css">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+1. Added index.html file.
+2. Added style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Web browser.
+2. Code editor.
+3. Git-smc.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/DanielTagua/microverse-project-porfolio-1
+```
+
+### Install
+
+Install this project with:
+
+
+Run this command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
+
+### Run tests
+>Coming soon
+
+### Deployment
+
```

Comment:
- [ ] Please add some text here because in your `README` file preview there is no information visible and it looks unprofessional to have a `header` without any information. Kindly add the information related to how any developer can use your project and deploy it on various free platforms, you can provide the information here.